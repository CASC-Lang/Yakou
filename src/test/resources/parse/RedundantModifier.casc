pub class RedundantModifier

impl RedundantModifier {
    pub new() {}

    pub fn a1() {}
}