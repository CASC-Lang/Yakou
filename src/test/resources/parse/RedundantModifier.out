error: Redundant access modifier `pub`, `pub` is declared under the hood by compiler
--> ./RedundantModifier.casc:1:0
1 | pub class RedundantModifier
    ^^^= hint: Remove this access modifier `pub`
2 |
error: Redundant access modifier `pub`, `pub` is declared under the hood by compiler
--> ./RedundantModifier.casc:4:4
3 | impl RedundantModifier {
4 |     pub new() {}
        ^^^= hint: Remove this access modifier `pub`
5 |
error: Redundant access modifier `pub`, `pub` is declared under the hood by compiler
--> ./RedundantModifier.casc:6:4
5 |
6 |     pub fn a1() {}
        ^^^= hint: Remove this access modifier `pub`